examples/simple-nft-module/src/lib.rs

```diff
@@ -36,6 +36,8 @@ impl<C: Context> Module for NonFungibleToken<C> {
 
     type CallMessage = CallMessage<C>;
 
+    type Event = ();
```

Comment:
Question:
How are we going to treat `EventKey`? Will it be always `Vec<u8>`, or should we allow any type to be the key?
cc @preston-evans98